docs/send-data/opentelemetry-collector/install-collector/linux.md

```diff
@@ -34,6 +34,14 @@ You can install our OpenTelemetry Collector using one of the following methods:
 * [Install script](#install-script)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+:::info
+Whitelist the following repository URLs to allow a seamless installation:
```

Comment:
We use "allowlist" and "denylist" instead of "whitelist" and "blacklist". https://help.sumologic.com/docs/contributing/style-guide/#cultural-neutrality

```suggestion
Allowlist the following repository URLs to allow a seamless installation:
```